docs/source/yelpsoa_configs.rst

```diff
@@ -281,12 +281,23 @@ instance MAY have:
     scripts will send a CRITICAL alert.
 
 In addition, each instancename MAY configure additional Marathon healthcheck
-options:
+options (Read the official
+`mesos documentation <https://mesos.readthedocs.io/en/latest/health-checks/>`_
+for more low-level details:
 
-  * ``healthcheck_mode``: One of ``cmd``, ``tcp``, ``http``, or ``https``. If your
-    service uses Smartstack, then this must match the value of the ``mode`` key
-    defined for this instance in ``smartstack.yaml``. If set to ``cmd`` then
-    PaaSTA will execute ``healthcheck_cmd`` and examine the return code.
+  * ``healthcheck_mode``: One of ``cmd``, ``tcp``, ``http``, or ``https``.
+    If set to ``http`` or ``https``, a ``curl`` command will be executed
+    inside the container. ``curl`` must be installed in the container for
+    this to work.
+
+    If set to ``cmd`` then PaaSTA will execute ``healthcheck_cmd`` and
+    examine the return code. It must return 0 to be considered healthy.
+
+    If the service is registered in SmartStack, he healthcheck_mode will
```

Comment:
s/he/the

Reply:
Damn by broken T key. I'm getting a replacement laptop soon.